mysos/scheduler/launcher.py

```diff
@@ -266,6 +272,13 @@ def _new_task(self, offer, task_cpus, task_mem, task_disk, task_port):
     task.name = task_id
 
     task.executor.executor_id.value = task_id  # Use task_id as executor_id.
+    task.executor.name = EXECUTOR_NAME
```

Comment:
This is akin to Aurora's executor name: `aurora.task` but yeah I guess it can simply be the task_id, which looks like `mysos-cluster123-0`.
Our internal monitoring tooling needs to use look at the executor name to distinguish it from tasks from other frameworks but we can certainly just match the prefix `mysos-`.